r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -16,6 +16,9 @@
 # under the License.
 
 skip_if_not_available("dataset")
+skip_if_multithreading_disabled()
+
+# withr::local_options(list(arrow.summarise.sort = TRUE))
```

Comment:
Should this be commented out here? I'm also a little surprised that the tests below pass since `arrange()` was removed.